src/components/Layout/Header/Header.tsx

```diff
@@ -7,11 +10,18 @@ const Header = () => {
       <div className={classes['c-header__logo']}>nQode</div>
       <nav className={classes['c-header__nav']}>
         <ul className={classes['c-header__nav-links']}>
-          <li className={classes['c-header__nav-link']}>Home</li>
-          <li className={classes['c-header__nav-link']}>Favourite</li>
-          <li className={classes['c-header__nav-link']}>Sales</li>
-          <li className={classes['c-header__nav-link']}>My books</li>
-          <li className={classes['c-header__nav-link']}>Profile</li>
+          {getAllowedNavbarLinks().map((link, index) => {
+            return (
+              <NavLink to={link.url} key={index} className={classes['c-header__nav-link']}>
+                {link.text}
+              </NavLink>
+            );
+          })}
```

Comment:
It is same for this part, you are mapping list of links but i don't think it is good idea